ReleaseChecklist.md

```diff
@@ -7,19 +7,38 @@
  - [ ] DockerHub account with push rights to the [``solc`` image](https://hub.docker.com/r/ethereum/solc).
  - [ ] Lauchpad (Ubuntu One) account with a membership in the ["Ethereum" team](https://launchpad.net/~ethereum) and
        a gnupg key for your email in the ``ethereum.org`` domain (has to be version 1, gpg2 won't work).
+ - [ ] Ubuntu/Debian dependencies of the PPA scripts: ``devscripts``, ``debhelper``, ``dput``, ``git``, ``wget``, ``ca-certificates``.
  - [ ] [npm Registry](https://www.npmjs.com) account added as a collaborator for the [``solc`` package](https://www.npmjs.com/package/solc).
  - [ ] Access to the [solidity_lang Twitter account](https://twitter.com/solidity_lang).
  - [ ] [Reddit](https://www.reddit.com) account that is at least 10 days old with a minimum of 20 comment karma (``/r/ethereum`` requirements).
 
+### Pre-flight checks
+At least a day before the release:
+ - [ ] Run ``make linkcheck`` from within ``docs/`` and fix any broken links it finds.
+       Ignore false positives caused by ``href`` anchors and dummy links not meant to work.
+ - [ ] Make sure that all merged PRs that should have changelog entries do have them.
+ - [ ] Rerun CI on the top commits of main branches in all repositories that do not have daily activity by creating a test branch or PR:
+      - [ ] ``solc-js``
+      - [ ] ``solc-bin`` (make sure the bytecode comparison check did run)
+      - [ ] ``homebrew-ethereum``
+ - [ ] Create a prerelease in our Ubuntu PPA by following the steps in the PPA section below on ``develop`` rather than on a tag.
```

Comment:
We really don't have to do that every time, if nothing related to it changed :-).

Reply:
Maybe not. But it uses an external service. There's always a chance that something will break and once we tag it may be too late to fix. I think it does not hurt to do a test run for this just in case.

Reply:
We're trying to make the release process smoother and quicker, not unnecessarily complicate it :-).

Reply:
Another reason is just to work through the process at least once and I think it's better not to do that on the release day. For me the PPA was actually one of the bigger hangups during the last release, and I think completing a pre-release before is one of the tion that would actually have made it smoother. You can always skip this step if you know there's absolutely nothing that can go wrong :)

Reply:
Then formulate it as optionally.

Reply:
ok, fine.

Reply:
Done.